releasenotes/notes/bump-ubuntu-snapshots-2023-09-15-22ca5250d40bd5b6.yaml

```diff
@@ -0,0 +1,7 @@
+---
+security:
+  - |
+    Bumps Ubuntu repository snapshots and container images to bring in latest
+    security patches. This includes the microcode to patch Inception
+    (CVE-2023-20569) and Downfall (CVE-2022-40982). Zenbleed (CVE-2023-20593)
+    was patched in the previous snapshot bump.
```

Comment:
Are these resolved via package update and reboot?

Reply:
That is the recommended approach. There is a feature called late loading, but this is not recommended:

https://docs.kernel.org/arch/x86/microcode.html#late-loading

Reply:
I've added a statement to say that the host needs rebooting.